metasrv/src/meta_service/raftmeta.rs

```diff
@@ -132,6 +134,7 @@ impl MetaNodeBuilder {
             running_tx: tx,
             running_rx: rx,
             join_handles: Mutex::new(Vec::new()),
+            joined_nodes: AtomicI32::new(1),
```

Comment:
It should be joined `tasks`, not `nodes`

Reply:
Done